docs/tutorials/11_quadratic_hamiltonian_and_slater_determinants.ipynb

```diff
@@ -0,0 +1,454 @@
+      "text/plain": [
+       "<Figure size 1544.38x264.88 with 1 Axes>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit_nature.circuit.library import FermionicGaussianState\n",
+    "\n",
+    "occupied_orbitals = (0, 2)\n",
+    "eig = np.sum(orbital_energies[list(occupied_orbitals)]) + transformed_constant\n",
+    "print(\"Eigenvalue:\")\n",
+    "print(eig)\n",
+    "\n",
+    "circuit = FermionicGaussianState(\n",
+    "    transformation_matrix, occupied_orbitals=occupied_orbitals\n",
+    ")\n",
+    "circuit.draw(\"mpl\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "498b232d-dde7-4248-aa61-be56a2b59a2d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit.quantum_info import Statevector\n",
+    "from qiskit_nature.mappers.second_quantization import JordanWignerMapper\n",
+    "\n",
+    "# simulate the circuit to get the final state\n",
+    "state = np.array(Statevector(circuit))\n",
+    "\n",
+    "# convert the Hamiltonian to a matrix\n",
+    "hamiltonian_jw = JordanWignerMapper().map(hamiltonian_ferm).to_matrix()\n",
+    "\n",
+    "# check that the state is an eigenvector with the expected eigenvalue\n",
+    "np.testing.assert_allclose(hamiltonian_jw @ state, eig * state, atol=1e-8)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e7d2a25e-5f5c-44ae-9e19-ff980704ed3a",
+   "metadata": {},
+   "source": [
+    "## Slater determinants\n",
+    "\n",
+    "When the antisymmetric part $\\Delta = 0$, then the Hamiltonian conserves particle number. In this case, the basis change only needs to mix creation operators, not annihilation operators:\n",
+    "$$\n",
+    "    \\begin{pmatrix}\n",
+    "    b^\\dagger_1 \\\\\n",
+    "    \\vdots \\\\\n",
+    "    b^\\dagger_N \\\\\n",
+    "    \\end{pmatrix}\n",
+    "    = W\n",
+    "    \\begin{pmatrix}\n",
+    "    a^\\dagger_1 \\\\\n",
+    "    \\vdots \\\\\n",
+    "    a^\\dagger_N \\\\\n",
+    "    \\end{pmatrix},\n",
+    "$$\n",
+    "where now $W$ is an $N \\times N$ matrix. Furthermore, the $\\{\\varepsilon_j\\}$ are allowed to be negative."
```

Comment:
If you name the `\epsilon_j` where you introduce them, referring to them by name
here will be easier to understand. Right now, the reader just sees a reference
to a set of variables which do not appear in the equation immediately above.

Reply:
Done.